### PR TITLE
Adding latest R-Instat help files

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons  these are the new help files uploaded by @rdstern . They could be merged as they have vbProj

File: instat/instat.vbproj [hunk 1]
```diff
@@ -5426,6 +5426,9 @@
     <Content Include="static\InstatObject\R\extras\bin\windows\contrib\3.3\zoo_1.7-14.zip">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="static\Help\R-Instat help.hnd" />
+    <None Include="static\Help\R-Instat.pdf" />
+    <None Include="static\Help\README.md" />
```
Comment (dannyparsons) [inline instat/instat.vbproj]: Do you think all of these extra files are needed? Also, the build action is None, is that correct?

Comment (maxwellfundi) [inline instat/instat.vbproj]: I just updated the already existing files with the new ones and so i think its useful to have them. I am not quite sure about build action being None.

**Comment by dannyparsons** [comment]
This has conflicts now on plot resources. Probably easiest is to redo this?

**Comment by maxwellfundi** [comment]
re-opening a new one pull request on this. This has conflicts so i have to close it

**Pull request closed without merging**